src/core/definitions.jl

```diff
@@ -27,6 +27,7 @@ const JuMPVariableMatrix = DenseAxisArray{
 const JuMPFloatArray = DenseAxisArray{Float64}
 const JuMPVariableArray = DenseAxisArray{JuMP.VariableRef}
 
+const TwoTerminalHVDCTypes = Union{PSY.TwoTerminalHVDCLine, PSY.TwoTerminalVSCDCLine}
```

Comment:
Should this be an abstract type?

Reply:
I don't think there is much value in an additional abstraction in PSY for this.